loader/loader.c

```diff
@@ -1455,6 +1468,32 @@ static VkResult loader_scanned_icd_add(const struct loader_instance *inst, struc
         icd_tramp_list->capacity *= 2;
     }
 
+    uint32_t major_version = VK_API_VERSION_MAJOR(api_version);
+    uint32_t minor_version = VK_API_VERSION_MINOR(api_version);
+    if (interface_vers <= 4 && 1 == major_version && 0 < minor_version) {
+        loader_log(inst, VULKAN_LOADER_ERROR_BIT, 0,
+                   "loader_scanned_icd_add: Driver %s says it supports api %u.%u, but only supports loader interface %u (Policy "
```

Comment:
Nit: The statement here isn't immediately clear about what is being violated. I know we shouldn't repeat the LDP_DRIVER_7 policy in full here, but first read through of this statement leaves me wondering why the Driver api version and the loader interface version have to do with eachother. Again, not a big deal, LDP_DRIVER_7 is very visible.

Reply:
I'll clean up the message.